src/services/videoconference/index.js

```diff
@@ -1,31 +1,38 @@
+const { promisify } = require('util');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [File has too many classes (4). Maximum allowed is 1.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5485544)